### DISCUSSION TITLE
I am desperate

### DISCUSSION THREAD

**Discussion by VolkmarMeyer** [shopware/frontends#81] [General]
Hi guys,
I am almost at the end.
what is the challenge? 
i am looking for a shopware developer for my company wigital gmbh who would like to work in a permanent position (no freelancer) in kiel.
**No, Kiel is not at the ass end of the world.** 

Is anyone between 0 and 1, all bits and bytes, online and willing to contact me? 

cheers 
Volkmar 

«Der Kopf ist rund, damit das Denken die Richtung wechseln kann»

**Comment by BrocksiNet** [comment]
Pls use this [slack channel](https://shopwarecommunity.slack.com/archives/C015XLNBWDC) to finde someone :+1:
